### PR TITLE
(CDAP-13342) Process metrics from TMS on SDK

### PR DESCRIPTION
**Pull request by chtyim**
This PR depends on #10204

### CODE REVIEW

**Comment by chtyim** [comment]
Build https://builds.cask.co/browse/CDAP-DUT6485

**Comment by chtyim** [comment]
Build passed https://builds.cask.co/browse/CDAP-DUT6485

File: cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseMetricsTableDefinition.java [hunk 1]
```diff
@@ -72,6 +91,17 @@ public DatasetSpecification configure(String name, DatasetProperties properties)
   @Override
   public MetricsTable getDataset(DatasetContext datasetContext, DatasetSpecification spec,
                                  Map<String, String> arguments, ClassLoader classLoader) throws IOException {
+    int datasetSplits = spec.getIntProperty(Constants.Metrics.METRICS_HBASE_TABLE_SPLITS, 16);
+
+    // Detect if there is a cdap-site change on the number of splits, which we don't support.
+    // Log a warning if that's the case.
+    if (cConf.getInt(Constants.Metrics.METRICS_HBASE_TABLE_SPLITS) != datasetSplits) {
+      CONFIG_CHANGE_LOG.warn(
+        "Ignoring {} configuration {} from cdap-site.xml because splits value can not be changed for system table {}",
```
Comment (poornachandra) [inline cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseMetricsTableDefinition.java]: Would be good to also log the splits value is being used instead.

**Review by poornachandra — COMMENTED** [review]
@chtyim Just one minor comment, please fix and merge

File: cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseMetricsTableDefinition.java (hunk 1 above)
Comment (chtyim) [reply cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseMetricsTableDefinition.java]: Updated message.

**Comment by chtyim** [comment]
Since the fix only involve changing a log message, merging without re-running the tests.

**Pull request merged**